modules/nw-multinetwork-sriov.adoc

```diff
@@ -312,3 +468,58 @@ $ oc create -f sriovsamplepod.yaml
 ----
 $ oc exec sriovsamplepod -- ip a
 ----
+
+User can also run RDMA or DPDK application in a pod with SR-IOV VF attached. The
+following is an example of a pod with a VF in RDMA mode:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rdma-app
+  annotations:
+    k8s.v1.cni.cncf.io/networks: sriov-rdma-mlnx
+spec:
+  containers:
+  - name: testpmd
+    image: <RDMA image>
+    imagePullPolicy: Never
+    securityContext:
+     capabilities:
+        add: ["IPC_LOCK"]
+    command: ["sleep", "infinity"]
+----
+
+This is an example of a pod with VF in DPDK mode:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dpdk-app
+  annotations:
+    k8s.v1.cni.cncf.io/networks: sriov-dpdk-net
+spec:
+  containers:
+  - name: testpmd
+    image: <DPDK image>
+    securityContext:
+     capabilities:
+        add: ["IPC_LOCK"]
+    volumeMounts:
+    - mountPath: /dev/hugepages
+      name: hugepage
+    resources:
+      requests:
+        memory: 1Gi
```

Comment:
Might be good to add `cpu` in resources requests and limits because dpdk application always need to bind cpu.

Reply:
resources:
      limits:
        memory: "1Gi"
        cpu: "2"
      requests:
        memory: "1Gi"
        cpu: "2"

Reply:
I updated to this:

```
    resources:
      requests:
        memory: "1Gi"
        cpu: "2"
        hugepages-1Gi: "4Gi"
      limits:
        hugepages-1Gi: "4Gi"
        cpu: "2"
```

Is that okay?

Reply:
> I updated to this:
> 
> ```
>     resources:
>       limits:
>         memory: "1Gi"
>         cpu: "2"
>         hugepages-1Gi: "4Gi"
>       requests:
>         memory: "1Gi"
>         cpu: "2"
>         hugepages-1Gi: "4Gi"
> ```
> 
> Is that okay? Or should `hugepages` be removed?

I think it's ok. DPDK application requires hugepages.

Reply:
Please add a note with link to kernel argument configurations (1G hugepage needs to be configured via kernel arguments, this config is supported via MCO). for example: https://github.com/openshift/openshift-docs/blob/enterprise-4.2/modules/nodes-nodes-kernel-arguments.adoc

Please also add a note for application using 2Mi hugepages (this doesn't require changing of kernel argument and can be configured via tuned Operator), link to the doc: https://github.com/openshift/openshift-docs/blob/enterprise-4.2/modules/configuring-huge-pages.adoc

Reply:
Adding kernel args: https://docs.openshift.com/container-platform/4.2/nodes/nodes/nodes-nodes-working.html#nodes-nodes-kernel-arguments_nodes-nodes-working